src/TextProvider.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export const TextContext = React.createContext<object>({});
+
+interface TextProviderProps {
+  globalText: any;
```

Comment:
`globalText` is an object of key/value as string/string. Can we add the definition here ?

Reply:
yes, changed it to `globalText: { [key: string]: string };`